<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FirstWindow</class>
 <widget class="QMainWindow" name="FirstWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>331</y>
      <width>461</width>
      <height>101</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="frame_rate">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>62</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="out_text_dir">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>161</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Output Files Directory</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_name">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>266</y>
      <width>551</width>
      <height>66</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Path to video</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="SubmitButton">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>95</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Initial Frame Rate</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>124</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Top Crop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="crop_top">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>121</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>154</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Bottom Crop</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="crop_bottom">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>219</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Shape</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>247</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cursor</string>
    </property>
   </widget>
   <widget class="QComboBox" name="shapeComboBox">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>212</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="maxCount">
     <number>10</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cursorComboBox">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>241</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="maxCount">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>191</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Annotation</string>
    </property>
   </widget>
   <widget class="QComboBox" name="annotationComboBox">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>184</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>4</number>
    </property>
    <property name="maxCount">
     <number>10</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
